concepts/nullability/.meta/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "blurb": "Learn about the null literal and nullable variables in java by printing name badges.",
```

Comment:
This blurb should be about the concept, not about the exercise. Could you try to think of a small blurb that just describes the `null` concept?